src/components/victory-scatter/victory-scatter.jsx

```diff
@@ -253,7 +272,37 @@ export default class VictoryScatter extends React.Component {
     return Helpers.evaluateStyle(baseDataStyle, data);
   }
 
-  renderPoint(data, index, calculatedProps) {
+  static getKeyedData(data) {
+    return data.reduce((keyedData, datum, idx) => {
```

Comment:
Let's have that as part of this PR, since there are more TODOs that definitely involve touching victory-core for animation anyway